opre_ops/ops_site/models.py

```diff
@@ -28,5 +31,43 @@ class Meta:
     def display_name(self):
         return self.first_name + " " + self.last_name
     display_name.short_description = "Full name"
-    
     full_name = property(display_name)
+
+    def __str__(self):
+        return self.full_name
+
+
+ARRANGEMENT_TYPES = [("1", "OPRE Appropriation"), ("2", "Cost Share"), ("3", "IAA"), 
+                    ("4", "IDDA"), ("5", "MOU")]
+class CANInfo(models.Model):
+    number = models.CharField(max_length=30)
+    description = models.CharField(max_length=100)
+    nickname = models.CharField(max_length=30)
+    purpose = models.TextField(null=True)
+    arrangement_type = models.CharField(max_length=30, choices=ARRANGEMENT_TYPES)
+    source = models.ManyToManyField(Agency)
+    authorizer = models.ForeignKey(Agency, on_delete=models.PROTECT, related_name="authorizer")
+
+    class Meta:
+        verbose_name_plural = "CANs"
+
+
+class CANAmount(models.Model):
```

Comment:
[_suggestion_]: Parallel suggestion to the one above ~ a docstring comment explaining what a CAN is, plus any other important information that a dev working on this model should know about the CANAmount model.